backend/src/cache/client_manager.go

```diff
@@ -110,7 +111,7 @@ func initMysql(params WhSvrDBParameters, initConnectionTimeout time.Duration) st
 		params.dbPort,
 		"",
 		params.dbGroupConcatMaxLen,
-		map[string]string{},
+		cast.ToStringMapString(params.dbExtraParams),
```

Comment:
Maybe it is better to do this vanilla instead of adding a new dependency: 

```
	var mysqlExtraParams = map[string]string{}
	data := []byte(params.dbExtraParams)
	json.Unmarshal(data, &mysqlExtraParams)
```
WDYT @maganaluis

Reply:
@NikeNano Certainly.